package.json

```diff
@@ -79,8 +77,8 @@
     "tslint-language-service": "^0.9.9",
     "typescript": "^2.9.2"
   },
-  "main": "dist/es6-es2015/src/index.js",
```

Comment:
We can discuss this change. I believe I used the es5 versions as a short-cut to avoid having to transpile this source elsewhere.